src/components/pages/donation_form/subpages/AddressPageDonationReceipt.vue

```diff
@@ -172,10 +165,10 @@ const store = useStore( StoreKey );
 
 const { addressType, addressTypeName } = useAddressType( store );
 const { addressSummary, inlineSummaryLanguageItem } = useAddressSummary( store );
-const { amount, interval, paymentType } = usePaymentValues( store );
 const mailingList = useMailingListModel( store );
 const { receiptNeeded, showReceiptOptionError } = useReceiptModel( store );
 const countryWasRestored = ref<boolean>( false );
+const submitValuesForm = ref<HTMLFormElement>();
```

Comment:
Rather creating this here than passing this into `useAddressFormEventHandlers` you can create it in there internally and return it with the `submit` and `previousPage` event handlers.